composer.json

```
@@ -26,7 +26,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ^8.0",
```

PHP does not follow [semantic versioning](https://semver.org) so please use:

```suggestion
        "php": "^7.3 || ~8.0.0 || ~8.1.0",
```